web/repos.jspf

```diff
@@ -171,8 +172,13 @@ Copyright (c) 2015, 2016, Oracle and/or its affiliates. All rights reserved.
                             if (branch == null) {
                                 branch = "N/A";
                             }
+                            String currentVersion = ri.getCurrentVersion();
+                            if (currentVersion == null) {
+                                currentVersion = "N/A";
+                            }
                             %><td><%= Util.htmlize(type) %></td><%
                             %><td><%= Util.htmlize(parent) %> (<%= Util.htmlize(branch) %>)</td><%
+                            %><td><%= Util.htmlize(currentVersion) %> </td><%
```

Comment:
When the commit message is quite long (which sometimes happen) the table looks ugly then - I'd rather see it on a single line.
I'd suggest something like displaying the date, changeset, author and first like 20-30 characters from the message (maybe with an option to show the whole message like it is in history view)

![screenshot from 2016-11-03 15-36-27](https://cloud.githubusercontent.com/assets/6997160/19970119/5bf6f1a6-a1db-11e6-9730-e69e4a2212b6.png)